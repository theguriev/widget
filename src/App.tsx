import React from 'react';
import { Widget } from './scenes';
import './assets/css/theme.css';

const App = () => <Widget />;

export default App;
