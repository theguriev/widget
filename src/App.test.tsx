import * as React from 'react';
import { render } from '@testing-library/react';

import App from './App';

describe('App', () => {
  it('renders app', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
