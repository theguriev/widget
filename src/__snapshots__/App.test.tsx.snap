// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders app 1`] = `
<DocumentFragment>
  <div
    class="widget"
  >
    <div
      class="list"
    >
      <button
        class="list"
        type="button"
      >
        <svg
          data-file-name="SvgSms"
        />
        SMS
      </button>
      <button
        class="list"
        type="button"
      >
        <svg
          data-file-name="SvgFacebookMessenger"
        />
        Messenger
      </button>
      <button
        class="list"
        type="button"
      >
        <svg
          data-file-name="SvgInstagram"
        />
        Instagram
      </button>
    </div>
    <button
      class="list startButton"
      type="button"
    >
      <svg
        data-file-name="SvgSms"
      />
      <svg
        data-file-name="SvgFacebookMessenger"
      />
      <svg
        data-file-name="SvgInstagram"
      />
      Message us
    </button>
  </div>
</DocumentFragment>
`;
